2023/typescript/lib.ts

```
@@ -0,0 +1,17 @@
+
+const INPUT_PATH = join(dir, '../', 'input')
+
+export default class Library {
+  static getInput(dayName: string): string {
+    return readFileSync(`${INPUT_PATH}/${dayName}.txt`).toString()
+  }
+
+  static getExampleInput(dayName: string): string {
+    return readFileSync(`${INPUT_PATH}/${dayName}-example.txt`).toString()
+  }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)